doc/user/content/guides/node-js.md

```diff
@@ -0,0 +1,245 @@
+---
+title: "Node.js and Materialize"
+description: "This guide explains how to use a Node.js application to connect, insert data, manage, and read data from Materialize."
+weight: 10
+menu:
+  main:
+    parent: guides
+---
+
+This guide explains all you need to know to start working with Materialize from a Node.js Application.
+
+Materialize is **PostgreSQL-compatible**, this means that a Node.js application can use any existing PostgreSQL client like `pg` ([node-postgres](https://node-postgres.com/)) to connect, manage, and read from Materialize as if it were a PostgreSQL database.
+
+## Connect
+
+Connecting to Materialize is the same as [connecting to PostgreSQL with `node-postgres`](https://node-postgres.com/features/connecting).
```

Comment:
I'll go with 
> You connect to Materialize the same way you [connect to PostgreSQL with `node-postgres`](https://node-postgres.com/features/connecting).

Because the thing I'm emphasizing is that **within node-postgres** it's the same as connecting to PostgreSQL.  There are other ways to connect to PostgreSQL from Node.JS besides `node-postgres` and 

>You connect to Materialize the same way you connect to PostgreSQL: by using node-postgres.

implies there is only one way.